rel/i18n/emqx_schema.hocon

```diff
@@ -1630,4 +1630,22 @@ client_attrs_init_set_as_attr {
     The extracted attribute will be stored in the `client_attrs` property with this name."""
 }
 
+banned_bootstrap_file.desc:
+"""The bootstrap file is a CSV file used to batch loading banned data when initializing a single node or cluster.
```

Comment:
The text suggests that it may be worth documenting the following:

The import action is only relevant or run when no banner in the database.

In other words, the import operation should only be performed if the banner being imported does not already exist in the database.